### PR TITLE
fix: wrong styled component

### PR DESCRIPTION
**Pull request by huydosgtech**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

**Review by hangnguyensaigontech — APPROVED** [review]

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ type BaseTouchableProps = Pick<
 >
 type StyledImageStyle = {
   theme?: ITheme
```
Comment (baonguyenhsts) [inline src/components/Checkbox/Checkbox.tsx:39]: remove this prop

File: src/components/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ type BaseTouchableProps = Pick<
...
+  tintColor?: string
 }
 type TextContainerStyle = {
   theme?: ITheme
```
Comment (baonguyenhsts) [inline src/components/Checkbox/Checkbox.tsx:43]: remove this prop

Comment (baonguyenhsts) [reply src/components/Checkbox/Checkbox.tsx:43]: Please check and remove all `theme?: ITheme`

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

**Review by honghoangsts — CHANGES_REQUESTED** [review]

**Review by ledutu2 — APPROVED** [review]

**Review by honghoangsts — APPROVED** [review]
